README.md

```diff
@@ -31,12 +31,17 @@ provided at the download links. You will need the Python 3.6 version.
 
 To install Devito, including examples, tests and tutorial notebooks, 
 follow these simple passes:
-```
+
+```sh
+# get the code
 git clone https://github.com/opesci/devito.git
 cd devito
-conda env create -f environment.yml
```

Comment:
I think both options should stay. Conda, in particular, should be the way to go for developers.

Reply:
Would keep conda too.

Reply:
Sounds good. I've restored the previous README state.